index.html

```diff
@@ -0,0 +1,772 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href=Css/Css-reset.css />
+  <link rel="stylesheet" href="Css/Style.css" />
+  <title>TNW-PREMA</title>
+</head>
+
+<body>
+  <section id="nav-sec">
+    <nav class="toprow-2">
```

Comment:
This should be a `header` tag.

Please read on how to use semantic HTML appropriately.